File: tests/data/cpp/working/json_in_cpp/Expected.cpp
json func() {
  return json{{"num", 0},
<<<<<<< LEFT
              {"list", json::array({{{"a", 1}, {"b", 2}}})},
||||||| BASE
              {{"num", 0},
              {"obj", {{"a", 1}, {"b", 2}}},
              {"list", json::array({{{"a", 3}, {"b", 4}}})}},
=======
              {{"num", 0},
              {"obj", {{"a", 1}, {"b", 2}}},
              {"list", json::array({{{"a", 3}, {"b", 4}}})},
              {"c", 5}},
>>>>>>> RIGHT
              {"other", {{"a", 3}, {"b", 4}}},
              {"c", 5}};
}

File: tests/data/spork/working/conflicting_operator_change/Expected.java
public class Adder {
    public int add(int a, int b) {
<<<<<<< LEFT
        return a * b;
||||||| BASE
        return a + b;
=======
        return a / b;
>>>>>>> RIGHT
    }

    public int otherOps() {
        int a = 1;
<<<<<<< LEFT
        a -= 1;
||||||| BASE
        a += 1;
=======
        a *= 1;
>>>>>>> RIGHT

<<<<<<< LEFT
        a = +a;
||||||| BASE
        a = ~a;
=======
        a = -a;
>>>>>>> RIGHT

        return a;
    }
}
